lib/metasploit/framework/data_service/remote/http/remote_module_data_service.rb

```diff
@@ -0,0 +1,12 @@
+require 'metasploit/framework/data_service/remote/http/response_data_helper'
+
+module RemoteModuleDataService
```

Comment:
It doesn't look like this is ever used. Is this just added in for future functionality?